ceremonies/src/lib.rs

```diff
@@ -310,11 +312,41 @@ decl_module! {
             <AttestationRegistry<T>>::insert((cid, cindex), &idx, &verified_attestation_accounts);
             <AttestationIndex<T>>::insert((cid, cindex), &sender, &idx);
             <MeetupParticipantCountVote<T>>::insert((cid, cindex), &sender, &claim_n_participants);
-            debug::debug!(target: LOG, 
-                "sucessfully registered {} attestations for {:?}", 
+            debug::debug!(target: LOG,
+                "sucessfully registered {} attestations for {:?}",
                 verified_attestation_accounts.len(), sender);
             Ok(())
         }
+
+        #[weight = 10_000]
+        pub fn endorse_newcomer(origin, cid: CurrencyIdentifier, newby: T::AccountId) -> DispatchResult {
+            debug::RuntimeLogger::init();
+            let sender = ensure_signed(origin)?;
+
+            // this is done to simplify the logic as a trade-off for UX, subject to change?
+            ensure!(<encointer_scheduler::Module<T>>::current_phase() == CeremonyPhaseType::REGISTERING,
+                "endorsing participants can only be done during REGISTERING phase");
+
+            ensure!(<encointer_currencies::Module<T>>::currency_identifiers().contains(&cid),
+                "CurrencyIdentifier not found");
+
+            ensure!(<encointer_currencies::Module<T>>::bootstrappers(&cid).contains(&sender),
+            "only bootstrappers can endorse newbies");
+
+            ensure!(<encointer_currencies::Module<T>>::bootstrapper_newby_tickets(&cid, &sender) > 0,
```

Comment:
newbie? is that wrong or both ok?

Reply:
Oops, you are right newby is incorrect, shall change that.